frontend/src/api.js

```diff
@@ -143,7 +153,9 @@ class Api {
       groupId,
       userId
     });
-  listGroup = () => instance.get(`/group.list`);
+  listGroup = () => {
```

Comment:
This can be changed back to arrow func without explicit return, probably there was logging in function body at some point.